web/src/components/projects/CreateProjectForm.tsx

```diff
@@ -109,29 +194,81 @@ export default function CreateProjectForm({ approvedStudies, handleProjectCreate
           </div>
 
           <div className={styles["form-field"]}>
-            <div className={styles["field-label"]}>Environment *</div>
-            <div className={styles["radio-group"]}>
-              <label className={styles["radio-option"]}>
-                <input
-                  type="radio"
-                  value="tre"
-                  {...register("environment", { required: "Please select an environment" })}
-                  disabled={isSubmitting}
-                />
-                <span>TRE (Trusted Research Environment)</span>
-              </label>
-
-              <label className={styles["radio-option-disabled"]}>
-                <input type="radio" value="dsh" disabled />
-                <span>DSH (Data Safe Haven) - Coming Soon</span>
-              </label>
-            </div>
-
-            {errors.environment && (
+            <label htmlFor="environmentId">Environment *</label>
+            <select
+              id="environmentId"
+              className={styles.select}
+              {...register("environmentId", {
+                required: "Please select an environment",
+              })}
+              disabled={isSubmitting || isLoadingEnvironments || !!environmentsError}
+            >
+              <option value="">
+                {isLoadingEnvironments
+                  ? "Loading environments..."
+                  : environmentsError
+                    ? "Error loading environments"
+                    : "Select an environment..."}
+              </option>
+              {environments.map((environment) => (
+                <option key={environment.id} value={environment.id}>
+                  {environment.name} (Tier {environment.tier})
```

Comment:
would be nice here to have the links to some documentation as some helper text once selected